app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,67 @@
+class FeedbacksController < ApplicationController
```

Comment:
why you have this controller. Do you follow the rails tutorial book?

Reply:
because i did all the chapter 6 coding without pushing the chapter 3 coding first

Reply:
may i know which part have not modify? because i double check all already, but still cannot find out

Reply:
please careful follow in the Rails tutorial book

Reply:
please delete this file because it not define on rails tutorial Book